geode-core/src/main/java/org/apache/geode/net/SSLParameterExtensionContextImpl.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.net;
+
+public class SSLParameterExtensionContextImpl implements SSLParameterExtensionContext {
+  private int distributedSystemId;
+
+  public SSLParameterExtensionContextImpl(int distributedSystemIdInput) {
+    distributedSystemId = distributedSystemIdInput;
+  }
+
+  @Override
+  public int getDistributedSystemId() {
+    return distributedSystemId;
+  }
+
+  public void setDistributedSystemId(int distributedSystemIdInput) {
+    distributedSystemId = distributedSystemIdInput;
+  }
```

Comment:
is it necessary for this to be a mutable class?  I don't see setDistributedSystemId called anywhere.  If the constructor and getter are sufficient, I would discourage a setter...

Reply:
At the moment, with only one value contained, there is not a compelling reason to need a setter. But I assume it will be expanded in the future to contain other context parameters for SSL, and that users will not want to pass (n) number or arguments into the constructor. Thus, a setter. Or we can over-engineer (IMO) and create a builder for a class containing one value.